src/compaction_execute_bench.rs

```diff
@@ -60,10 +60,12 @@ pub fn run_compaction_execute_bench() -> Result<(), SlateDBError> {
     let options = load_options();
     let s3 = open_object_store(&options)?;
     let sst_format = SsTableFormat::new(4096, 1, options.compression_codec);
+    let block_cache = Some(Arc::new(BlockCache::new(1 << 30))); // 1GB block cache,
```

Comment:
can we use the size based eviction rather than the entry based? Entry based won't keep us below the desired size threshold if we use the cache for indexes/filters.

Reply:
SGTM. I think moka comes with a size-based policy as well.

https://docs.rs/moka/latest/moka/future/struct.Cache.html#example-size-based-eviction